src/components/RequestTile.svelte

```diff
@@ -32,23 +33,49 @@ h3.smaller {
 .request-tile > .request-image.smaller {
   height: 6rem;
 }
+.small-caps {
+  font-variant: small-caps;
+}
 .smaller {
   font-size: 0.9rem;
 }
+.smaller .from-prefix,
```

Comment:
would you be able to simply use bs's `d-block` class and eliminate these customizations?

Reply:
Maybe... the trick is that these need to only be `display: block` when their parent element has the (optional) `smaller` class applied. They aren't always `displaly: block`.